src/Microsoft.PowerShell.Commands.Management/resources/NavigationResources.resx

```diff
@@ -168,6 +168,9 @@
   <data name="MoveItemInUse" xml:space="preserve">
     <value>Cannot move item because the item at '{0}' is in use.</value>
   </data>
+  <data name="RenameMultipleItemError" xml:space="preserve">
+      <value>Cannot rename item because the path '{0}' resolved to multiple items. Only one item can be renamed at a time.</value>
```

Comment:
It makes no sense to tell the user that we have tried to resolve wildcards - in fact, we consider the Path parameter as literal.

Reply:
I wonder if we would add support for renaming multiple items like the `ren` command in cmd?
eg. ```Rename-Item -Path '*.txt' -NewExtension '.bak'```
or, ```Rename-Item -Path '*.txt' -Format 'renamed_{0}'```

Reply:
I believe the enhancement is out of the PR and we should discuss it in a new tracking Issue (please open).